doc/requirements/io_class

```diff
@@ -0,0 +1,85 @@
+---
+group: IO class
+---
+
+Requirements for IO classes
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting occupancy limit
+id: occupancy_per_ioclass
+---
+
+It shall be possible to set a maximum size of an IO class.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Occupancy limit interpretation
+id: occupancy_per_ioclass_limit_interpretation
+---
+
+The product shall shall interpret max occupancy value as a percentage of
+a cache size
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Allowed occupancy values
+id: occupancy_per_ioclass_allowed_values
+---
+
+The product shall accept values for a max size of an IO class in range from 0 to 100
```

Comment:
I am not sure if it should be mentioned here or somewhere else, but in the IO class config file this value can be set only to values between 0.0-1.0, not  0-100

Reply:
sooo true - this also changes the interpretation req

Reply:
and default limit